pkg/objectio/column.go

```diff
@@ -183,3 +194,19 @@ func (cb *ColumnBlock) UnMarshalMate(cache *bytes.Buffer) error {
 	}
 	return err
 }
+
+func (cb *ColumnBlock) freeData(entry fileservice.IOEntry, m *mpool.MPool) {
+	if m != nil {
+		m.Free(entry.Data)
+	}
+}
+
+func (cb *ColumnBlock) allocData(entry fileservice.IOEntry, m *mpool.MPool) (err error) {
```

Comment:
This code, if m is nil, it will not allocate data?   I don't see how this will work.

if m == nil { panic}?

Reply:
if m == nil { fileservice will make buffer to fill data}

Reply:
file service will make buffer?   Where is the buffer stored?   How can you tell if you need to free the buffer or not?